.github/workflows/e2e_tests.yml

```diff
@@ -94,6 +94,8 @@ jobs:
           GH_USERNAME: ${{ secrets.GH_USERNAME }}
           GOV1_PHRASE: ${{ inputs.gov1_mnemonic || secrets.GOV1_PHRASE }}
           GOV2_PHRASE: ${{ inputs.gov2_mnemonic || secrets.GOV2_PHRASE }}
```

Comment:
not directly related to this PR but we should remove these inputs for gov1 and gov2 and get the phrases from secrets only